routing.md

```diff
@@ -277,9 +277,35 @@ Why is such indirection useful?
 * It lets the mediator remain uncommitted to whether the next receiver is another mediator or not. This may provide flexibility in some routing scenarios.
 * It lets the mediator change the size of the message by adding or subtracting noise from the content. 
 * It allows for dynamic routing late in the delivery chain.
- 
+
 These last two characteristics are the foundation of mix networking feature for DIDComm. That feature is the subject of a different RFC; here we only note the existence of the optional feature. 
 
 
-### DID Docs and Routing
+### DID Document Service Endpoint
+DIDComm DID Document endpoints have the following format:
+
+```json
```

Comment:
To have maximum interoperability, I suspect that we should consider a JSON-LD context hosted from DIF that contains custom vocabulary.

Reply:
Agree. After we decide on the needed attributes, a PR to include such information would be useful.

Reply:
Sounds good - seems like a followup topic.